robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+
+Resource       robot/pmm/resources/pmm.robot
+Library        cumulusci.robotframework.PageObjects
+...            robot/pmm/resources/ServiceSchedulePageObject.py
+...            robot/pmm/resources/ServicePageObject.py
+Suite Setup     Run Keywords
+...             Open Test Browser
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
```

Comment:
Could you update the suite setup to be able to run the test as non admin users?

Suite Setup     Run Keywords
...             Open test browser            useralias=${test_user}             AND
...             Setup Test Data
Suite Teardown  Capture Screenshot and Delete Records and Close Browser

*** Variables ***
${test_user}             UUser

*** Keywords ***

Reply:
Thank you, adding this!

Reply:
@rmonica08 - Added those lines and the updated keyword file.